### PR TITLE
Junelee/alter design/#115

### PR DESCRIPTION
**Pull request by leeejune**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #115  (issue, if applicable)

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._
Changed spacing for all types of lists and bullet size.

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by leeejune** [comment]
@braven112 I made some changes here!

File: src/utilityClasses/_listProperties.scss [hunk 1]
```diff
@@ -180,7 +165,7 @@ li::before {
 /// @example scss - import mixin file
 ///   @import "./node_modules/@aurodesignsystem/webcorestylesheets/dist/utilityClasses/listProperties";
 #{$scope}.#{$prefix}util_listAuroIcon {
-  padding-left: 2*$auro-size-200;
+  padding-left: 2*$auro-size-200; //calc(x*x)
```
Comment (braven112) [inline src/utilityClasses/_listProperties.scss:168]: We can remove this comment and use the calc function to generate this value..

File: src/utilityClasses/_listProperties.scss [hunk 2]
```diff
@@ -180,7 +165,7 @@ li::before {
 /// @example scss - import mixin file
 ///   @import "./node_modules/@aurodesignsystem/webcorestylesheets/dist/utilityClasses/listProperties";
 #{$scope}.#{$prefix}util_listAuroIcon {
-  padding-left: 2*$auro-size-200;
+  padding-left: 2*$auro-size-200; //calc(x*x)
   list-style: none;
   li {
     text-indent: -$auro-size-200;
```
Comment (braven112) [inline src/utilityClasses/_listProperties.scss:171]: This should be a css variable instead

**Comment by blackfalcon** [comment]
I removed the merge from @braven112 but accidentally removed the last two commits from @leeejune. I reached out to @leeejune to force-push her last version of the branch so that we can have that work. Then I can rebase off of main and merge this work.

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Requires missing commits from @leeejune

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**